cernopendata/modules/records/descriptions.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Open Data Portal.
+# Copyright (C) 2017 CERN.
+#
+# CERN Open Data Portal is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Open Data Portal; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Descriptions for supporting collections pages for CERN Open Data Portal."""
```

Comment:
- Since these are not really describing records, but collections, perhaps they should live in `cernopendata/modules/collections/descriptions.py` rather than `cernopendata/modules/records/descriptions.py`?  I guess you did not want to create a new module, but it may be safer?

- A collection description may perhaps have a picture associated.  Shall we aim at more flexible description structure already?

Reply:
Something like the following perhaps:

```
collections = {
  'cms': { 'name': 'CMS',
               'description': 'Foo bar blah....',
               'logo': '/static/cms/.png'
             },
}
```

I think we don't need to keep the notion of ancestor tree, since all collections were directly attached under one of the four experiments.

Otherwise LGTM.
```